content/assets/blog/phoenix-case-study/index.md

```diff
@@ -0,0 +1,137 @@
+what data to capture, helping them focus on the most relevant content while honouring platform and
+data privacy policies.
+
+### **Classification systems for all languages**
+
+A standout feature is Phoenix’s multilingual classification engine, which manages high- and
+low-resource languages alike. This ensures peacebuilders can quickly identify posts of
+interest—crucial when sifting through huge volumes of potentially conflict-related material.
+
+***
+
+## **How it works**
+
+1. **Gather**: Peacebuilders specify keywords, locations, or other criteria for collecting data
+   from social media and other sources.
+
+2. **Classify**: Phoenix’s machine learning models process the incoming data, identifying critical
+   topics, trends, and sentiments.
+
+3. **Explore**: Users start with a standard dashboard and tailor visualizations to track regional
+   tensions or other priorities.
+
+4. **Collaborate & share**: Being open-source, Phoenix allows contributors worldwide to enhance its
+   capabilities—ensuring continued evolution and impact.
+
+***
+
+## **The results: A global impact on peacebuilding**
+
+Since launch, Phoenix has proven invaluable for peacebuilders worldwide, yielding several
+noteworthy outcomes:
+
+1. **Global reach**: Versions v1 and v2 of Phoenix have supported more than 50 peacebuilders,
+   allowing them to analyze social media discourses and develop targeted interventions. Including
+   being one of the 31 of 770 projects accepted into the [Paris Peace
+   Forum](https://www.elysee.fr/en/sommet-pour-l-action-sur-l-ia/2024/10/24/call-for-projects-from-the-paris-peace-forum-a-growing-interest-in-addressing-global-challenges-through-ai).
+
+2. **Massive data handling at low cost**: Phoenix has processed terabytes of social media data,
+   managing the scale more commonly associated with large tech firms than small nonprofits.
+
+3. **Real insights**: By revealing the narratives driving online discussion, Phoenix enables
+   quicker, more evidence-based decision-making. Case studies can be found in the [Phoenix
+   manual](https://www.notion.so/howtobuildup/Case-studies-cd8ace41995a479a820c3f722532440c).
+
+4. **Public good platform launch**: Scheduled for 2025, Phoenix will be offered as a public-good
+   platform, bringing large-scale data analytics to even more communities and organizations. Read
+   more [here](https://howtobuildup.org/phoenix/).
+
+## **Conclusion**
+
+Phoenix stands as a transformative example of how collaboration and user-centric innovation can
+address complex global challenges. By combining Build Up’s peacebuilding expertise with
+datavaluepeople’s technical prowess, the platform empowers mediators to analyze social media data
+at scale, uncovering critical insights that inform conflict prevention strategies. Its ability to
+process terabytes of multilingual data, deliver real-time visualizations, and adapt to diverse
```

Comment:
"adapt to diverse regions" isn't meaningful when we already have "multi-lingual" in the sentence

Reply:
51f9a89